openapi/components/schemas/RiskScore/RiskScoreBlocklistType.yaml

```
@@ -0,0 +1,8 @@
+type: object
+properties:
+  riskScoreThreshold:
+    type: integer
+  ttl:
+    type: integer
+    description: Blocklist TTL. Defaults to zero, meaning blocklist record won't expire ever.
```

nitpick: suggestion to add more detail
```suggestion
    description: |-
      Length of time a blocklist exists before it expires.
      The default field value for this field is `0`, which means that the blocklist record never expires.
```